### PR TITLE
feat: TypeScript alpha build

### PR DESCRIPTION
**Pull request by alexander-fenster**
Adding TypeScript targets. At this moment:
- the generator is taken from [`diregapic`](https://github.com/googleapis/gapic-generator-typescript/tree/diregapic) branch
- the generated code [depends](https://github.com/googleapis/gapic-generator-typescript/blob/diregapic/templates/typescript_gapic/package.json#L40) on `google-gax` from [`transcoding`](https://github.com/googleapis/gax-nodejs/tree/transcoding) branch (which makes installation very slow)
- no support for pagination and LRO at this time, it's work in progress

Usage:
```sh
bazel build //google/cloud/compute/v1:compute-small-v1-nodejs   # full compute also works, small is faster
```

Unpack the resulting tarball, prepare the library:
```sh
mkdir compute-library-nodejs
cd compute-library-nodejs
tar xzf /path/to/googleapis-discovery/bazel-bin/google/cloud/compute/v1/compute-small-v1-nodejs.tar.gz
cd compute-small-v1-nodejs
npm install
npm pack  # will produce google-cloud-compute-small-0.1.0.tgz
```

Run the sample:
```
mkdir compute-sample-nodejs
cd compute-sample-nodejs
npm init -y
npm install /path/to/compute-library-nodejs/compute-small-v1-nodejs/google-cloud-compute-small-0.1.0.tgz
curl -L https://gist.githubusercontent.com/alexander-fenster/fbaac2b8e3cb734a81c85238aeefe32f/raw/eda612ea86b3fd98a0d9ecaba17eda451addf8b6/index.js > index.js
export GOOGLE_APPLICATION_CREDENTIALS=/path/to/key.json
node index.js
```

Sample code: [gist](https://gist.github.com/alexander-fenster/fbaac2b8e3cb734a81c85238aeefe32f)

### CODE REVIEW

File: google/cloud/compute/v1/BUILD.bazel [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# This file was automatically generated by BuildFileGenerator
+#o This file was automatically generated by BuildFileGenerator
```
Comment (vchudnov-g) [inline google/cloud/compute/v1/BUILD.bazel:1]: nit: type `#o`

Comment (alexander-fenster) [reply google/cloud/compute/v1/BUILD.bazel:1]: Oops. Fixed :)

File: WORKSPACE [hunk 2]
```diff
@@ -288,16 +289,11 @@ go_gapic_repositories()
...
-_gapic_generator_typescript_sha256 = "1206afaf44143de21881c59646ce6a121f490b8ef193fc81f00bae20e90dbb7d"
-
 ### TypeScript generator
 http_archive(
     name = "gapic_generator_typescript",
-    sha256 = _gapic_generator_typescript_sha256,
-    strip_prefix = "gapic-generator-typescript-%s" % _gapic_generator_typescript_version,
-    urls = ["https://github.com/googleapis/gapic-generator-typescript/archive/v%s.tar.gz" % _gapic_generator_typescript_version],
+    strip_prefix = "gapic-generator-typescript-diregapic",
+    urls = ["https://github.com/googleapis/gapic-generator-typescript/archive/diregapic.tar.gz"],
```
Comment (vam-google) [inline WORKSPACE:296]: Please depend on a specific commit.

Comment (alexander-fenster) [reply WORKSPACE:296]: Done

**Review by vam-google — APPROVED** [review]
LGTM

**Pull request merged**